src/charm.py

```diff
@@ -429,7 +238,7 @@ def _configure_ingress(self) -> None:
                 self._update_tracing_relations()
 
         # notify the cluster
-        self._update_tempo_cluster()
+        self.coordinator.update_cluster()
```

Comment:
@lucabello 
as you see here we need to call update_cluster like it were a common exit hook after a bunch of different events.
Would it make sense to add to Coordinator something like a `cluster_refresh_events` list of events that the coordinator?

The only caveat to this approach would be that we'd need to call update_cluster AFTER the event processing has executed, on commit, because otherwise the framework will queue that before the subclass' own handlers.

Reply:
No preference really, I'd say let's see what happens in all the three HA solutions, and we can refactor it afterwards, maybe?

Calling the function is a bit more "explicit", but we can do whichever